Nimble/Expectation.swift

```diff
@@ -1,5 +1,25 @@
 import Foundation
 
+func expectTo<T, U where U: Matcher, U.ValueType == T>(expression: Expression<T>, matcher: U, to: String) -> (Bool, FailureMessage) {
```

Comment:
I definitely agree. I think I forgot this one, but added it other places. I'll add `internal`.

:+1: For a consistent style guide. I don't really care too much what the conventions are, just that it's consistent.